src/Mover.ts

```diff
@@ -763,6 +799,90 @@ export class MoverAPI implements Types.MoverAPI {
         }
     };
 
+    private _focusMoverFor(
+        focusedElement: HTMLElement | undefined,
+        keyCode: number
+    ): void {
+        // Mover's nextFor property allows a Mover to gain focus when
+        // the arrow keys are pressed when the focus is currently not
+        // inside any Mover. Here we find if there is a Mover with
+        // nextFor matching the currently focused element and focus it (if there is).
+
+        const moversFor = this._moversFor;
+        const allMoversFor: { element: HTMLElement; mover?: Mover }[] = [];
+        const focusedElementOrBody =
+            focusedElement || this._win().document.body;
+
+        for (const nextFor of Object.keys(moversFor)) {
+            if (matchesSelector(focusedElementOrBody, nextFor)) {
+                for (const mover of moversFor[nextFor]) {
+                    const element = mover.getElement();
+
+                    if (element) {
+                        allMoversFor.push({ element, mover });
+                    }
+                }
+            }
+        }
+
+        if (allMoversFor.length > 0) {
```

Comment:
you don't need the extra sort right? you could just check for the closest mover while you are matching selectors.